src/components/Registration/BusinessTypeConfirm.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div id="business-type-confirm">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3" class="pr-4">
+        <label class="d-block">Business Type</label>
+        <v-chip
+          v-if="hasBusinessTypeChecked"
+          id="checked-chip"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          Checked
+        </v-chip>
+      </v-col>
+
+      <v-col cols="12" sm="9">
+        <p class="mb-0 checkbox-gp-text">{{ label }}</p>
+        <div id="business-check-div">
+          <v-checkbox
+            class="mt-0"
+            v-model="checked"
+            hide-details
+            :label="text"
+          >
+          </v-checkbox>
+        </div>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
+
+@Component
+export default class BusinessTypeConfirm extends Vue {
+  /** The registration business number. */
+  @Prop({ required: true })
+  readonly businessTypeConfirm!: boolean
+
+  /** Whether to display Change features. */
+  @Prop({ default: false })
+  readonly hasBusinessTypeChecked!: boolean;
+
+  /** Whether the business typ is SP or GP */
+  @Prop({ default: false })
+  readonly isTypePartnership!: boolean;
```

Comment:
Are you sure this component displays for SP? My understanding was that this component is for GP only.

(I know the designs show both, but I think those are old designs 🤦‍♂️ )

Reply:
OK, it looks like Yui left some comments in the ticket fairly recently so I guess those are the "latest".

Please update the ticket description to call out the SP/DBA version. (Delete "old description" and just clean it up.)

Reply:
Yes. I've updated the description in Zenhub. Here is the [link](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/9828) fyi.